backend/minors/CGS.yaml

```diff
@@ -0,0 +1,132 @@
+and anthropology. The interdisciplinary character of cognitive science is 
+reflected in its reliance on many levels of analysis and its employment of 
+a variety of methodologies appropriate to each level. The goal of cognitive 
+\science is to integrate the insights from multiple disciplines and modes of
+research into a unified scientific account of minds – be they human or 
+non-human, natural or artificial – and their place in nature and society.
+contacts:
+- type: Program Director
+  name: Anna Colasante
+  email: anna@princeton.edu
+req_list:
+  - name: Total Courses
+    explanation: "Students are required to take five courses in cognitive science"
+    min_needed: 5
+    max_counted: 1
+    req_list:
+    - name: Required Courses
+      explanation: At least one course designated with an asterisk in the list below, indicating that it is a CogSci Minor Required Course (CMRC);
+      min_needed: 1
+      max_counted: 1
+      course_list: Required Courses
+        - CHV 300
+        - CGS 322
+        - CGS 254
+        - CGS 255
+        - COS 360
+        - LIN 309
+        - MUS 248
+        - NEU 306 
+        - PHI 207
+        - PHI 322
+        - PSY 248
+        - PSY 254
+        - PSY 255
+        - PSY 306
+        - PSY 309
+        - PSY 315
+        - PSY 333
+        - PSY 360
+        - PSY 454
+    - name: Advanced Courses
+      explanation: "Three courses at the 300-level or higher."
+      min_needed: 3
+      max_counted: 3
+      course_list:
+        - COS 126
+        - COS 324 
+        - COS 402
+        - COS 424
+        - COS 429
+        - COS 436
+        - COS 484
+        - COS 495
+        - LIN 201 
+        - LIN 205 
+        - LIN 219 
+        - LIN 250 
+        - LIN 301 
+        - LIN 302
+        - LIN 303 
+        - LIN 306
+        - LIN 308
+        - LIN 310
+        - LIN 355
+        - LIN 360 
+        - LIN 406 
+        - LIN 412
+        - MAE 345
+        - ENE 475
+        - ECE 364
+        - NEU 175
+        - NEU 200
+        - NEU 201
+        - NEU 202  
+        - NEU 202B
+        - NEU 330 
+        - NEU 437
+        - PHI 207
+        - PHI 218
+        - PHI 311
+        - PHI 313
+        - PHI 315
+        - PHI 317
+        - PHI 321
+        - *PHI 322
+        - PHI 352
+        - PHI 380
+        - *PSY 254
+        - *PSY 255
+        - PSY 304
+        - *PSY 306
+        - PSY 307
+        - *PSY 309
+        - *PSY 315
+        - PSY 316
+        - *PSY 333
+        - PSY 337
+        - PSY 338 
+        - PSY 340
+        - PSY 345
+        - *PSY 360
+        - PSY 400
+        - PSY 409
+        - *PSY 454
+        -  SPI 305
+        - SPI 340
+        - SPI 404
+        - ECO 468
+        - *MUS 248
+        - HUM 230
+        - TRA 301
+        - HIS 494
+        - SML 354
+  - name: Academic Units
+    explanation: "Courses must be taken in at least three different academic units."
```

Comment:
Let us give "Academic Units" subreqs by departments with `min_needed` = 3 and `max_counted` = 1. Make sure we have double counting true